pwiz_tools/Skyline/Model/AuditLog/AuditLogStrings.resx

```diff
@@ -632,8 +632,8 @@
   <data name="modified_outside_of_skyline" xml:space="preserve">
     <value>The document was modified outside of Skyline: {0}</value>
   </data>
-  <data name="AuditLogList_Validate_Audit_log_is_corrupted__Audit_log_entry_time_stamps_and_indices_should_be_decreasing" xml:space="preserve">
-    <value>Audit log is corrupted. Audit log entry time stamps and indices should be decreasing</value>
+  <data name="AuditLogList_Validate_Audit_log_is_corrupted__Audit_log_entry_time_stamps_and_indices_should_be_decreasing_Entry_timestamp__0_G___entry_order_number" xml:space="preserve">
+    <value>Audit log is corrupted. Audit log entry time stamps and indices should be decreasing. Entry timestamp {0:G}, entry order number {1}.</value>
```

Comment:
I would improve this message to give all the information in the message without requiring the user to look into the file to understand exactly what is going on.

Audit log is corrupted. Audit log entry timestamps and indices should be decreasing.

Entry {0} timestamp {1:G} occurs before entry {2} timestamp {3:G}.

or

Entry {0} appears after entry {1}.

Depending on whether the issue is out of order dates or indices.

You can use TextUtil.LineSeparate() to combine the two messages.